crates/sargon/src/keys_collector/collector/derivation_purpose.rs

```diff
@@ -12,11 +12,15 @@ pub enum DerivationPurpose {
     /// for identity VECIs
     CreatingNewPersona,
 
-    /// When applying a security shield to an account, initiates keys collection
+    /// When applying a security shield to accounts and personas mixed, initiates keys collection
+    /// for account MFA
+    SecurifyingAccountsAndPersonas,
```

Comment:
@micbakos-rdx we will automatically select this as `DerivationPurpose` if we are securifying both account **and** personas mixed - but as we have talked about in the past, host probably maps all keys to the same copy.